modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 
 require 'ddtrace'
-require 'simple_forms_api_submission/service'
 require 'simple_forms_api_submission/metadata_validator'
-require 'simple_forms_api_submission/s3'
```

Comment:
This deletion is unrelated and should have been done earlier when I removed CHAMPVA code from this controller.